src/index.html

```diff
@@ -1,17 +1,245 @@
 <!doctype html>
 <html lang="en">
+
   <head>
-    <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    charset="UTF-8"
+    >
+    <meta
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0,
+    maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
```

Comment:
Please fix the formatting of the code. You have to use the correct code indentations.

Look Dorota's mentions before and apply it in the whole code :)

```
    <form 
       action="https://mate-academy-form-lesson.herokuapp.com/create-application"
       method="post"
       name="personalInformation"
    >
```